level2/data.tf

```diff
@@ -1,8 +1,20 @@
 data "terraform_remote_state" "level1" {
   backend = "s3"
   config = {
-    bucket = "terraform-remotestate3215"
+    bucket = "terraform-remotestate-abc123xyz"
     key    = "level1.tfstate"
     region = "us-east-1"
   }
 }
+data "aws_ami" "amazonlinux" {
+  most_recent = true
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-kernel-*"]
+  }
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+  owners = ["137112412989"]
+}
```

Comment:
could you please, add a newline at the end of file.

Reply:
Done.